docs/api/routing.rst

```diff
@@ -374,6 +374,54 @@ A custom router is any class that implements the following interface:
 
             """
 
+Suffixed Responders
+-------------------
+
+While Falcon encourages the REST architectural style, it is flexible enough to accomodate other architectures. Consider the task of building an API 
+for a calculator which can both add add subtract two numbers. You could implement the following:
+
+.. code:: python
+
+    class Add(object):
```

Comment:
nitpick. `class Foo(object): ...`  is a bit outdated, and no longer required in python3

Reply:
Ah, fair, updated